### PR TITLE
Use brighter emoji tick/cross icons in docs

### PR DESCRIPTION
**Pull request by edg2s**

### CODE REVIEW

File: rule-tester-and-docs.js [hunk 1]
```diff
@@ -73,13 +73,13 @@ class RuleTesterAndDocs extends RuleTester {
...
-			output += '✗ The following patterns are considered errors:\n' +
-				buildRuleDetails( tests.invalid, '✗' );
+			output += '❌ The following patterns are considered errors:\n' +
+				buildRuleDetails( tests.invalid, '❌' );
 
 			output += '\n';
 
-			output += '✓ The following patterns are not considered errors:\n' +
-				buildRuleDetails( tests.valid, '✓' );
+			output += '✔️ The following patterns are not considered errors:\n' +
```
Comment (jdforrester) [inline rule-tester-and-docs.js]: Maybe use ✅ instead?

Comment (edg2s) [reply rule-tester-and-docs.js]: The other one renders as a green tick in GitHub, complementing the red 'x'

Comment (jdforrester) [reply rule-tester-and-docs.js]: > The other one renders as a green tick in GitHub, complementing the red 'x'

Only on your browser/OS, not on mine:

image

Comment (edg2s) [reply rule-tester-and-docs.js]: I mean the one that is in the commit renders green (although it won't in the diff). GitHub (where most people will read the docs?) replaces emoji with images, so it shouldn't be browser-dependent. Here's how the current patch looks for me on the site:

image

Comment (jdforrester) [reply rule-tester-and-docs.js]: Oh, yeah, if I look at the "rich diff" it does indeed look better.

**Pull request merged**